pkg/lib/listing.scss

```diff
@@ -85,7 +85,6 @@
 tbody > tr.listing-ct-item {
     border: 1px solid transparent;
     border-top: 1px solid var(--color-ct-list-border);
-    border-bottom: 1px solid var(--color-ct-list-border);
```

Comment:
I guess you want to remove both border-top and border-bottom lines here?

Reply:
We do want border lines between the cells, but I'll check.